src/server/game/Handlers/QuestHandler.cpp

```diff
@@ -608,15 +608,24 @@ void WorldSession::HandlePushQuestToParty(WorldPacket& recvPacket)
 
         sender->SendPushToPartyResponse(receiver, QUEST_PARTY_MSG_SHARING_QUEST);
 
+        bool questAutoAccepted = false;
         if (quest->IsAutoAccept() && receiver->CanAddQuest(quest, true) && receiver->CanTakeQuest(quest, true))
+        {
             receiver->AddQuestAndCheckCompletion(quest, sender);
+            questAutoAccepted = true;
+        }
```

Comment:
```suggestion
        if (quest->IsAutoAccept() && receiver->CanAddQuest(quest, true) && receiver->CanTakeQuest(quest, true))
        {
            receiver->AddQuestAndCheckCompletion(quest, sender);
            sender->SendPushToPartyResponse(receiver, QUEST_PARTY_MSG_ACCEPT_QUEST);
            return;
        }
```

Reply:
@Treeston does that still show the quest gossip ?

Reply:
Oh, should it? I thought auto-accept just went straight to log.